gdx/src/com/badlogic/gdx/graphics/glutils/ETC1.java

```diff
@@ -114,10 +129,13 @@ public void write (FileHandle file) {
 					write.write(buffer, 0, bytesToWrite);
 					writtenBytes += bytesToWrite;
 				}
+
+				write.close();
+
+				gzip.close();
+
 			} catch (Exception e) {
-				throw new GdxRuntimeException("Couldn't write PKM file to '" + file + "'", e);
-			} finally {
-				StreamUtils.closeQuietly(write);

```

Comment:
No problem, we're all always learning something new.

Btw. `os.close();` on 107 is now redundant and the finally should be probably used in `write(OutputStream)` as well.